Backend/index.js

```diff
@@ -113,33 +113,41 @@ app.get('/api/feed', async (req, res, next) => {
     }
 })
 
-//Add friend : {username, friendUsername}
+//follow : {username, followUsername}
 // takes in a query with above fields and returns a JSON Success or throws a Backend Error
-app.get('/api/addfriend', async (req, res, next) => {
+app.post('/api/follow', async (req, res, next) => {
     try {
-        let { username, friendUsername } = req.query;
```

Comment:
minor but prolly better to stay consistent with naming, ie use `friend_username`